web/client/components/widgets/builder/wizard/common/wfsChartOptions.js

```diff
@@ -28,13 +23,13 @@ const getAllowedAggregationOptions = (propertyName, featureTypeProperties = [])
 };
 
 export default compose(
-    withProps(({featureTypeProperties = [], data = {}} = {}) => ({
-        options: propsToOptions(featureTypeProperties),
-        typedOptions: propsToTypedOptions(featureTypeProperties),
```

Comment:
here we remove `typedOptions` but it seems still used by https://github.com/geosolutions-it/MapStore2/pull/9151/files#diff-3a08ea6a0593652ea1a7b46fd9d30c287b9c105b8e54a70b2c7fcdf4787f82ffR314 , is this prop passed from another part?